fix: return `null` for missing user and group fields in VFolder lookup API response (#2584)
lablup/backend.ai: changes/2584.fix.md

> The VFolder lookup API endpoint `GET /func/folders/{folderName}` now correctly returns `null` for `user` and `group` fields when they are absent, instead of returning the string "None".

Reviewer: ```suggestion
The VFolder lookup API `GET /func/folders/{folderName}` now returns `null` for missing `user` and `group` fields, instead of returning `None` as string.
```

Reviewer: Let's refine the changelog. The others look good to me.
